app/models/metric/ci_mixin/processing.rb

```diff
@@ -98,24 +94,12 @@ def perf_process(interval_name, start_time, end_time, counters_data)
       publish_metrics(rt_rows) if syndicate_metrics?
     end
     _log.info("#{log_header} Processing for #{log_specific_targets(resources)}, for range [#{start_time} - #{end_time}]...Complete - Timings: #{t.inspect}")
-
-    affected_timestamps
```

Comment:
I had changed to:

```diff
- affected_timestamps
+ affected_timestamps.uniq
```

question: Is this was intentional.
answer: yes

new answer: `affected_timestamps` is not used, so dropping it